cmd/collector/app/builder/span_handler_builder.go

```diff
@@ -54,23 +54,23 @@ type SpanHandlerBuilder interface {
 }
 
 // NewSpanHandlerBuilder returns a span handler
-func NewSpanHandlerBuilder(opts ...basicB.Option) (SpanHandlerBuilder, error) {
+func NewSpanHandlerBuilder(cOps *CollectorOptions, flagOpts *flags.Flags, opts ...basicB.Option) (SpanHandlerBuilder, error) {
```

Comment:
see https://github.com/uber/jaeger/pull/235/commits/bba2c5ac1bcaceca216ea80c05c82ec462172162#diff-1e05e6cc5af73dff91ea191eba47978dL32
I'm not sure whether I did it properly